roles/provision-sync-app-apb/tasks/main.yml

```diff
@@ -1,11 +1,16 @@
-- name: Config Map Template
-  template:
-    src: config.yaml.j2
-    dest: /tmp/foo.yaml
-
 - name: create configmap
-  shell: oc create configmap -n "{{ namespace }}" "{{ _apb_service_instance_id }}" --from-file=/tmp/foo.yaml
-
+  k8s:
+    state: present
+    definition: 
+      apiVersion: v1
+      kind: ConfigMap
+      data:
+        SYNC_APP_NAME: '{{ SYNC_APP_NAME }}'
+      metadata:
+        name: '{{ _apb_service_instance_id }}'
+        namespace: '{{ namespace }}'
+        labels:
+          app: sync-app-{{ SYNC_APP_NAME }}
```

Comment:
Great idea 👍 
Btw should be [the template](https://github.com/aerogearcatalog/sync-app-apb/tree/5957f60c07591fba03d676e05080d26c9e8fc9a3/roles/provision-sync-app-apb/templates) removed when we are not using it anymore?

Reply:
I removed it!